groups/prometheus/main.tf

```diff
@@ -0,0 +1,76 @@
+provider "aws" {
+  region = var.region
+}
+
+terraform {
+  backend "s3" {}
+}
+
+module "security-groups" {
+  source           = "./module-security-groups"
+  region           = var.region
+  vpc_id           = local.vpc_id
+  ssh_cidrs        = concat(local.internal_cidrs,local.vpn_cidrs)
```

Comment:
Add a space after each argument to the `concat` function to separate them visually as it makes identifying the different arguments easier at a glance:

```yml
concat(local.internal_cidrs, local.vpn_cidrs)

```